actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,31 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :@variants
```

Comment:
For what it's worth, I kept looking and found two apps using #== in the first 15 pages of Ruby code search results on GitHub:

* [codemy/blogmenow](https://github.com/codemy/blogmenow/blob/17a248fd9b3a28dca146e0957b31f90f51237bd7/app/controllers/application_controller.rb)
* [cassiuschen/bdfzer-sso](https://github.com/cassiuschen/bdfzer-sso/blob/2ca76a5aec9e9f9267955202d0c3058496afccd6/app/controllers/home_controller.rb)

Not sure if this changes whether we should leave #== in. It's clearly not widely used, but it makes me nervous that it could silently change behavior for the few apps that do rely on it.